kitsune/wiki/templates/wiki/translate.html

```diff
@@ -171,7 +171,7 @@
             <div class="approved">
               <h3>{{ _('Approved {default_locale} version:')|f(default_locale=settings.LANGUAGES_DICT[settings.WIKI_DEFAULT_LANGUAGE.lower()]) }}</h3>
               <div id="content-or-diff" class="content">
-                <textarea readonly="readonly">{{ based_on.content }}</textarea>
+                <textarea readonly="readonly" dir="ltr">{{ based_on.content }}</textarea>
```

Comment:
Better yet, lets use the setting `settings.LANGUAGES_BIDI`, which is a list of the locale codes that are RTL. It is provided to us by Django's i18n.